extern/guided_diffusion/guided_diffusion/train_util.py

```diff
@@ -43,6 +44,7 @@ def __init__(
         self.model = model
         self.diffusion = diffusion
         self.data = data
+        self.val_data: DataLoader = val_data
```

Comment:
should we leave this type hinting? maybe consider moving the hints to the arguments instead.

Reply:
I prefer being inconsistently better than consistently bad